common/src/main/java/com/wynntils/features/inventory/PersonalStorageUtilitiesFeature.java

```diff
@@ -34,6 +38,12 @@
 
 @ConfigCategory(Category.INVENTORY)
 public class PersonalStorageUtilitiesFeature extends Feature {
+    @Persisted
+    private final Config<CustomColor> SelectedQuickJumpColor = new Config<>(CommonColors.GREEN);
```

Comment:
```suggestion
    private final Config<CustomColor> selectedQuickJumpColor = new Config<>(CommonColors.GREEN);
```

Make sure to apply this change to where it is used as well

Reply:
Well... that's kinda embarrassing...